src/main/java/gov/usgs/wma/statistics/model/JsonDataBuilder.java

```diff
@@ -89,7 +92,13 @@ public JsonDataBuilder latestValue(String value) {
 	}
 	
 	public JsonDataBuilder mediation(MediationType mediation) {
-		this.mediation = mediation;
+		// This logic allows for a setting override of a mediation
+		// since the most prevalent mediation might undesirable.
+		// The statistics will use the default sort if none is given,
+		// usually the default will be ascending for non-well samples.
+		if (MediationType.DEFAULT.equals(this.mediation)) {
+			this.mediation = mediation;
+		}
```

Comment:
This default logic is preparation for later work. We need to be able to override the mediation while using the most prevalent if not overridden. Again, default prevents the need for a null and NPE.